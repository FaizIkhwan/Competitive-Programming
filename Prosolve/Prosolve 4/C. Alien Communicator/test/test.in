3
good morning
how are you
do you come in peace