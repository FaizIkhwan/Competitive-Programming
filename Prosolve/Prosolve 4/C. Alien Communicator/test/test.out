morning good
you are how
peace in come you do
